# author: Samuel Moors (Vrije Universiteit Brussel)
# Fix CmakeLists.txt to avoid Boost error:
# CMake Error at .../Boost/1.71.0-gompi-2019b/lib/cmake/Boost-1.71.0/BoostConfig.cmake:182 (boost_find_component):
#   boost_find_component Macro invoked with incorrect arguments for macro
#   named: boost_find_component
# Call Stack (most recent call first):
#   .../CMake/3.15.3-GCCcore-8.3.0/share/cmake-3.15/Modules/FindBoost.cmake:443 (find_package)
#   CMakeLists.txt:363 (find_package)
diff -ur dakota-6.11.0.src.orig/CMakeLists.txt dakota-6.11.0.src/CMakeLists.txt
--- dakota-6.11.0.src.orig/CMakeLists.txt       2019-11-08 01:19:53.000000000 +0100
+++ dakota-6.11.0.src/CMakeLists.txt    2022-09-01 16:38:30.349794048 +0200
@@ -360,7 +360,7 @@
   set(Boost_USE_STATIC_LIBS TRUE)
 endif()
 # Dakota requires Boost 1.49 or newer; enforce for all libs in the build
-find_package(Boost 1.49 REQUIRED COMPONENTS "${dakota_boost_components}")
+find_package(Boost 1.49 REQUIRED COMPONENTS ${dakota_boost_components})
 # This cache variable is used to package the Boost liraries on Darwin (see
 # cmake/InstallDarwinDylibs.cmake). The variable Boost_LIBRARY_DIRS is set
 # in this scope by the CMake Boost probe (at least for CMake 3.5.2), but